doc/1.4/language.md

```diff
@@ -987,22 +987,43 @@ Templates are imported into an object declaration body using
 <pre>
 is <em>name</em>;
 </pre>
+for example:
+```
+field F {
+    is A;
+}
+```
 
 It is also possible to use templates when declaring an object, as in
 
 <pre>
 field F is (<em>name1</em>, <em>name2</em>);
 </pre>
 
-These can be used in any context where an object declaration may be
-written, and has the effect of expanding the body of the template at
-the point of the `is`. If two templates define methods or
-parameters with the same name, then the template instantiation
-hierarchy is used to deduce which method overrides the other: If one
-template *B* instantiates another template *A*, directly or indirectly,
-then methods from *B* override methods from *A*. Note, however, that
-overrides can only happen on methods and parameters that are
-declared `default`. Example:
+These can be used in any context where an object declaration may be written, and
+has the effect of expanding the body of the template at the point of the `is`.
+Using `is` together with object declarations is typically more idiomatic than
+the standalone `is` object statement; however, the latter is useful in order
+to instantiate templates for the top-level device object, and also for use in
+conjunction with [`in each` declarations](#in-each-declarations); for example:
+
+```
+register r {
+    in each field {
+        is A;
```

Comment:
doesn't hurt to concretize, e.g. `is write_1_clears;` turns it into a potentially real use case